modules/plugin/jdbc/jdbc-db2/src/main/java/org/geotools/data/db2/DB2NGDataStoreFactory.java

```diff
@@ -75,6 +80,24 @@
                     "Use selectivity clause for spatial queries",
                     false,
                     Boolean.TRUE);
+    /* the following values are helpful to pre-set for testing - comment out for release */
```

Comment:
This seems like a bad idea. You should provide a properties option, or do this in the test code.

Reply:
Agreed, the values should be in a property file driving the test, like we do for other databases, see here:
https://github.com/geotools/geotools/blob/master/.github/workflows/postgis_online.yml#L37

No manual action takes place during releases.

Reply:
I'm not clear on where the property file driving the test is located or how it is setup. Currently, automatic testing of online functionality is not possible.

Is it possible to use a Docker image for the online testing?

How are the Oracle JDBC driver files handled?

Reply:
It looks like my pull request is due to the test Oracle XE failing. How do I get around this?

Reply:
When reviewing we go in and check why the build failed, in this case it is not due travis (and not any change you made in this PR we are reviewing).